src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartReferenceLinesConfiguration.tsx

```diff
@@ -134,7 +136,34 @@ export default function ChartReferenceLinesConfiguration({
             }}
             validationSchema={Yup.object<AddFormValues>({
               label: Yup.string().required('Enter label'),
-              position: Yup.string().required('Enter position'),
+              otherAxisPosition: Yup.number().test({
+                name: 'otherAxisPosition',
+                message: 'Enter a valid position',
+                test: (value: number) => {
+                  if (!value) {
```

Comment:
If the value is 0, this validation will pass, even if it's actually out of bounds on the other axis. Note that users can change the minor axis' min/max, so we need to cover the possibility of this not being 0.

I'm guessing this is supposed to cover the 'blank' case, so we should really do something like this instead:

```ts
test: (value?: number) {
  if (typeof value !== 'number) {
    return true;
  }

  // do validation...
}
```